### ISSUE TITLE
Invalid self-referencing anchor created in absence of wgPatentURI

### ISSUE THREAD

**Issue by SuyashSalampuria** [w3c/respec#2638]
## Important info

* URL to affected spec: https://github.com/w3c/respec/blob/develop/src/w3c/templates/sotd.js

Invalid self-referencing anchor are created if there is no conf.wgPatentURI value set. 
Replacing `[conf.wgPatentURI]` with `conf.wgPatentURI` gives an error in 
`npm run test:headless`

**Comment by ridhishjain** [comment]
@SuyashSalampuria can u please describe how to reproduce this behavior?

**Comment by saschanaz** [comment]
I think it's already self-explanatory, focus on this part:

>Replacing [conf.wgPatentURI] with conf.wgPatentURI gives an error in
`npm run test:headless`

**Comment by saschanaz** [comment]
@marcoscaceres, should this be an error?

**Comment by marcoscaceres** [comment]
Yes, this should definitely be an error. I wonder also why it's in "[]"... not checked, but that seems odd.
